src/assets/data/localisations.ts

```diff
@@ -93,6 +93,16 @@ export const Localisations = {
     hb:
       'לחץ על כפתור הסריקה וכוון את המצלמה לכיוון קוד הQR שניתן לך על ידי החוקרים. דוגמה לקוד QR מוצגת למטה.'
   },
+  ENROL_REGISTRATION_ORY_DESC: {
+    da: 'Log ind med din Ory identitetsudbyder.',
+    de: 'Melden Sie sich mit Ihrem Ory-Identitätsanbieter an.',
+    en: 'Login with your Ory identity provider.',
```

Comment:
this can be removed after changes suggested above (better to avoid any technical jargon like Ory for participants)